content/faq.md

```diff
@@ -11,3 +11,7 @@ metadata into a high fidelity graph database—normalizing entity identities and
 mapping standard relationships between them. Querying this graph can drive
 higher-level organizational outcomes such as audit, policy, risk management, and
 even developer assistance.
+
+### What SPDX versions are supported
+
+GUAC supports [version 2.3](https://spdx.github.io/spdx-spec/v2.3/) of the SPDX specification.
```

Comment:
Based on slack message, I had this wrong. Looks like the library supports all v2.x

Reply:
Updated to reflect this information.